apps/nextjs-website/src/components/Feature.tsx

```diff
@@ -25,10 +26,56 @@ const makeFeatureProps = ({
   ...rest,
 });
 
-const Feature = (props: FeatureSection) => (
-  <section id={props.sectionID || undefined}>
-    <FeatureEC {...makeFeatureProps(props)} />
-  </section>
-);
+const Feature = (props: FeatureSection) => {
+  const theme = useTheme();
+
+  return (
+    <section id={props.sectionID || undefined}>
+      <Stack
+        sx={{
+          section: {
+            ...(props.theme === 'light' && { backgroundColor: '#FAFAFA' }), // Using value found in SEND website over Figma
```

Comment:
Avoid hard-coded values. Use theme color or create custom theme. E.g.: Use `background.default`, or create a new theme and customize the palette color:

```typescript
'use client';
import { createTheme } from '@mui/material/styles';
import { theme as muiItaliaTheme } from '@pagopa/mui-italia';

export const theme = createTheme(muiItaliaTheme, {
  palette: {
    background: {
      default: '#FAFAFA',
    },
  },
});
```

The above code should be placed into `/apps/nextjs-website/src/app/theme.ts` file, and it should be imported by [/apps/nextjs-website/src/app/layout.tsx#L24](https://github.com/pagopa/b2b-portals/blob/135790f/apps/nextjs-website/src/app/layout.tsx#L24) file instead the `mui-italia` theme.

Reply:
Creating a whole new theme for just one slightly different backgroundColor value (FAFAFA over F2F2F2) seems excessive.

I'm just going to utilize MUI Italia's value if hardcoded values are unwelcome to avoid overcomplicating.

It would also just be hiding the "hardcoded-ness" behind the use of `theme` since the value is currently not used anywhere else.